### PR TITLE
WIP: move info about trainings in general to /about

### PR DESCRIPTION
**Pull request by ksuess**
- markdown /MyST instructions for authors
- instructions to setup sphinx, etc locally
- how to prepare and give a training: communication

### CODE REVIEW

**Comment by stevepiercy** [comment]
Please do not move any files at this time. It will break the conversion process. After conversion to MyST, then we can move files and content around. Otherwise they will not get included in the conversion. Sorry for the inconvenience, but it means we do not have to hit a moving target.

For now you may make any _changes_ to content in `.rst` files in my open PR #512.

Any _conversions_ of `.rst` to `.md` should be postponed until after #512 and #511 are approved and merged.

**Comment by ksuess** [comment]
Yes, thats OK for me. It' not meant to fiddle with the current master. I am just preparing some info for future authors : myst / markdown, … And while writing I thought this info and also the sphinx setup should not stay in mastering_ but would be better placed in a folder independent of an individual training. The PR sits on top of the theme PR. So everything's fine.

**Comment by stevepiercy** [comment]
We should discuss where we move information that is not an actual training. I have opinions :wink:. Let's collect notes in Discord to discuss during our meeting on Thursday.

**Comment by stevepiercy** [comment]
I'm not sure where this should go, but this is the only thing that generates warnings in Sphinx.

It might need some indentation and polish.

---

```markdown
## Syntax highlighting in code blocks

When including code snippets, you should specify the language.
Possible languages are listed on the [Pygments lexers page](https://pygments.org/docs/lexers/).
The snippet _must_ be valid syntax for the language you specify, else it will not be highlighted.
The following example will highlight the code snippet as a shell, regardless of flavor.

    ```shell
    source env/bin/activate
    ```

### Language tips

-   Use `shell` for commands issued in a terminal session.
    Do not include shell prompts.
-   Use `console` for output of a shell session.
    If you have a mix of a shell command and its output, then use `console`.
-   If `xml` does not work well, then try `html`.
-   If no language works, then try `text`.

### Validate syntax

Always build the page to validate syntax.
Your own training should not be merged into the larger Training docs if there are any Sphinx warnings.
The Sphinx console will display any warnings, such as the following.

    ```console
    /Plone/training/voltohandson/introtoblocks.md:55: WARNING: Could not lex literal_block as "jsx". Highlighting skipped.
    ```

The above warning indicates that the syntax is not valid.
Common mistakes include:
-   Using `...` to indicate omitted code.
    If you must do that, comment it out using the language's comment syntax.
-   Using comments in JSON.
-   A previous code block bleeds through to the next due to invalid MyST syntax.

If you are unable to resolve the warning by fixing syntax, then use an all-purpose language such as `text` or `console`.
At least then the build will succeed without warnings, although syntax highlighting for such snippets will not appear.

As of this writing, the following files have syntax highlighting warnings.

```text
angular/aggregating.md:150: html+ng2
angular/aggregating.md:207: html+ng2
angular/aggregating.md:227: scss
angular/footer.md:59: scss
angular/overriding.md:56: scss
angular/overriding.md:173: scss
mastering-plone-5/restapi.md:18: http
mastering-plone-5/restapi.md:40: http
mastering-plone/dexterity_reference.md:780: js
mastering-plone/events.md:266: less
mastering-plone/restapi.md:18: http
mastering-plone/restapi.md:40: http
mastering-plone/volto_actions.md:260: jsx
mastering-plone/volto_components_sponsors.md:270: jsx
mastering-plone/volto_talk_listview.md:57: jsx
mastering-plone/volto_talkview.md:62: jsx
solr/a-fancy-custom-search-page.md:131: http
theming/adv-diazo.md:66: less
theming/theme-package-3.md:284: less
volto/custom-widget.md:18: jsx
volto/custom-widget.md:1: jsx
volto/i18n.md:140: jsx
voltoaddons/06-cell-renderer.md:199: jsx
voltohandson/blocksedit.md:318: less
voltohandson/blocksmainslider.md:119: less
voltohandson/contenttypesviews.md:17: jsx
voltohandson/contenttypesviews.md:91: less
voltohandson/footer.md:15: jsx
voltohandson/footer.md:66: less
voltohandson/header.md:41: less
voltohandson/header.md:113: js
voltohandson/highlightsblock.md:3: jsx
voltohandson/highlightsblock.md:15: jsx
voltohandson/highlightsblock.md:110: less
voltohandson/highlightsblock.md:224: jsx
voltohandson/introtoblocks.md:43: jsx
voltohandson/introtoblocks.md:55: jsx
```

**Comment by ksuess** [comment]
Needs to be rebased later.

File: about/instructions.md [hunk 1]
```diff
@@ -18,12 +18,26 @@ meta data
...
+
+  If you have a page at myfolder/mypage.md (relative to your documentation root) with the following structure:
+
+      # Title
+
+      ## My Subtitle
+
+  Then the autosectionlabel feature will allow you to reference the section headers like so:
+
+      {ref}`path/to/file_1:My Subtitle`
```
Comment (stevepiercy) [inline about/instructions.md:32]: Will we have to rewrite all our references to use this syntax, or will they be OK as is?

**Comment by ksuess** [comment]
closed in favour of https://github.com/plone/training/pull/524

This text from Steve https://github.com/plone/training/pull/513#issuecomment-912169849 should be integrated. I added a link to the overall issue https://github.com/plone/training/issues/506

**Pull request closed without merging**